hardware/diskimages.rst

```diff
@@ -271,3 +271,100 @@ Supported formats:
+
+Creating floppy images
+**********************
+
+The following command will create a 1.4M (1440K, double-sided, 18 sectors per track, 80 cylinders) floppy image named ``floppy.img`` with a label of ``LABEL``:
+
+.. code-block::
+
+  mformat -f 1440 -v LABEL -C -i floppy.img ::
+
+The ``-f`` option specifies the format of the floppy being created. The command can be adjusted for format, label, and image name as needed.
+
+Please see the `mtools documentation <https://www.gnu.org/software/mtools/manual/mtools.html#mformat>`_ for more information on the supported formats.
+
+Copying files to floppy images
+******************************
+
+The following command will copy ``file1`` and ``file2`` to the floppy image ``floppy.img``:
+
+.. code-block::
+
+  mcopy -i floppy.img file1 file2 ::
+
+Wildcards are also supported with ``mcopy``.
+
+.. note:: The ``::`` is required to let ``mtools`` know there are no more files to copy or arguments to process.
+
+CD-ROM: macOS
+^^^^^^^^^^^^^
+
+macOS can natively mount CD-ROM ISOs, but to create them you'll need to open up the terminal.
+
+The following command creates an ISO file named ``cdrom.iso`` with the volume name ``CDROM``.
+
+.. code-block::
+
+  hdiutil makehybrid -iso -joliet -joliet-volume-name "CDROM" -o /path/to/cdrom.iso /path/to/cd/root
+
+In the above example the directory ``/path/to/cd/root`` becomes the root directory of the ISO image.
+
+If you wanted your current working directory to be the ISO root filesystem you could use the following command:
+
+.. code-block::
+
+  hdiutil makehybrid -iso -joliet -joliet-volume-name "CDROM" -o ../cdrom.iso .
+
+.. note:: Make sure the output filename with ``-o`` has a path outside of the filesystem root.
+
+CD-ROM: Linux
+^^^^^^^^^^^^^
+
+Linux provides the ``mkisofs`` tool in order to easily create ISO images. The following command creates the ISO file ``cdrom.iso`` which contains the contents of the directory ``/path/to/cd/root``:
+
+.. code-block::
+
+  mkisofs -o cdrom.iso /path/to/cd/root
+
+.. note::
+  This package is available in the standard distribution repositories, generally under the ``mkisofs`` or ``genisoimage`` package names.
+
+Disk Image: macOS
+^^^^^^^^^^^^^^^^^
+
+macOS can natively mount raw disk images (floppy or hard disk) of types ``FAT16`` and ``FAT32``. Simply double click the file in Finder to mount the image.
+
+For fixed-size ``vhd`` files the following command may work depending your your macOS version:
+
+.. code-block::
+
+  hdiutil attach -imagekey diskimage-class=CRawDiskImage /path/to/your/vhd
+
+.. note:: As with any image file in macOS, the image can only be mounted if macOS can read the underlying filesystem. macOS can read both ``FAT16`` and ``FAT32``.
+
+Various: Windows
+^^^^^^^^^^^^^^^^^
+
+On Windows you can use WinImage (paid software) to create and manipulate disk images.
```

Comment:
```suggestion
On Windows you can use WinImage to create and manipulate disk images.
```

WinImage really falls into the same category of "paid" software as WinRAR 😛

Reply:
You're right. Some projects are super picky about this so I was playing it safe.